python-sdk/src/astro/lineage/__init__.py

```diff
@@ -1,7 +1,25 @@
+import os
+
 import logging
 
+from airflow.configuration import conf
+
 log = logging.getLogger(__name__)
 
+
+def _is_disabled() -> bool:
+    return (
+        conf.getboolean("openlineage", "disabled", fallback=False)
+        or os.getenv("OPENLINEAGE_DISABLED", "false").lower() == "true"
+        or (
+            conf.get("openlineage", "transport", fallback="") == ""
+            and conf.get("openlineage", "config_path", fallback="") == ""
+            and os.getenv("OPENLINEAGE_URL", "") == ""
+            and os.getenv("OPENLINEAGE_CONFIG", "") == ""
+        )
+    )
```

Comment:
This code is copy from https://github.com/apache/airflow/blob/390eacb01a710239e69f71d0c52134b7e87d17d4/airflow/providers/openlineage/plugins/openlineage.py#L27-L37